blocks/research/__paragraph/research__paragraph.css

```diff
@@ -0,0 +1,14 @@
+.research__paragraph {
+  font-size: 18px;
+  line-height: 30px;
+  color: rgb(39, 53, 68);
+  padding: 25px 0 30px 0;
```

Comment:
@Woolfs1423 The same idea here: let's create that spacing between our title and paragraph using margin instead of padding. 

- reset `research__paragraph`'s padding to 0
- use _shorthand notation_ to assign an element with a top margin equal to `25px` - for desktop and `10px` - for tablet and keep everything else `0`.